cowamm/cow_amm_cow_with_users_surplus_query_4031724.sql

```diff
@@ -19,38 +18,92 @@ with cow_amm_surplus as (
     where istrade
 ),
 
-cow_surplus_per_batch as (
+cow_surplus_per_batch_ethereum as (
```

Comment:
Shouldn't we also add arbitrum?

Maybe we should keep the current query that takes blockchain as an arg and gives us `solver_name`, `total_cow_surplus_in_usd` and then have the first query in this file be something like


```
with 
aggregate_results_per_solver_all_chains as (
    select
        solver_name,
        sum(total_cow_surplus_in_usd) as total_cow_surplus_in_usd
    from (
         select * from queryX(blockchain='ethereum')
         union all
         select * from queryX(blockchain='gnosis')
         union all
         select * from queryX(blockchain='arbitrum')
     )
    group by solver_name
),
```